exercises/practice/square-root/src/main/groovy/SquareRoot.groovy

```diff
@@ -0,0 +1,7 @@
+class SquareRoot {
+
+    static Integer squareRoot(Integer radicand) {
+        throw new UnsupportedOperationException('Move implementation is missing')
```

Comment:
Should this text be updated?

Reply:
Whoops, I thought I had. I’ll take care of it.